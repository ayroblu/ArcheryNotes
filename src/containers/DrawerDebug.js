// @flow
import React, { Component } from 'react'
import {
  StyleSheet,
  Text,
  View,
  TouchableOpacity,
  Platform,
} from 'react-native'
import {connect} from 'react-redux'
import {bindActionCreators} from 'redux'

import DevTools from '../components/DevTools'
import {textStyle} from '../config'
import * as theme from '../theme'

class DrawerDebug extends Component {
  render() {
    return (
      <DevTools />
    )
  }
}

const styles = StyleSheet.create({
})

export default connect(state=>({
  main: state.main
}))(DrawerDebug)
