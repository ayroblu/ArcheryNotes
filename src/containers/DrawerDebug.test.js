import 'react-native'
import React from 'react'
import {Provider} from 'react-redux'
import DrawerDebug from './DrawerDebug'
import configureStore from '../store'
Date.now = jest.fn(() => 1487076708000)

// Note: test renderer must be required after react-native.
import renderer from 'react-test-renderer'

const store = configureStore()

it('renders correctly', ()=>{
  const props = {
  }
  const tree = renderer.create(
    <Provider store={store}>
      <DrawerDebug {...props} />
    </Provider>
  )
  expect(tree).toMatchSnapshot()
})
