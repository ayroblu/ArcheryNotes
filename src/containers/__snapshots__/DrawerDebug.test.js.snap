// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<View
  style={
    Object {
      "backgroundColor": "#333",
      "flex": 1,
      "paddingTop": 20,
    }
  }
>
  <RCTScrollView>
    <View>
      <View
        style={
          Object {
            "margin": 10,
          }
        }
      >
        <Text
          accessible={true}
          allowFontScaling={true}
          disabled={false}
          ellipsizeMode="tail"
          style={
            Object {
              "alignSelf": "flex-end",
              "color": "white",
              "fontSize": 12,
              "fontWeight": "300",
              "textAlign": "right",
            }
          }
        >
          2017-07-15T23:08:54.530Z
        </Text>
        <Text
          accessible={true}
          allowFontScaling={true}
          disabled={false}
          ellipsizeMode="tail"
          style={
            Object {
              "color": "white",
              "fontSize": 18,
              "fontWeight": "300",
            }
          }
        >
          0
          : 
          @@INIT
        </Text>
        <View>
          <Text
            accessible={true}
            allowFontScaling={true}
            disabled={false}
            ellipsizeMode="tail"
            style={
              Object {
                "color": "white",
                "fontSize": 14,
                "fontWeight": "300",
              }
            }
          >
            { type: '@@INIT' }
          </Text>
          <View
            style={
              Object {
                "bottom": 0,
                "flexDirection": "row",
                "left": 0,
                "position": "absolute",
                "right": 0,
                "top": 0,
              }
            }
          >
            <View
              accessibilityComponentType={undefined}
              accessibilityLabel={undefined}
              accessibilityTraits={undefined}
              accessible={true}
              hasTVPreferredFocus={undefined}
              hitSlop={undefined}
              isTVSelectable={true}
              nativeID={undefined}
              onLayout={undefined}
              onResponderGrant={[Function]}
              onResponderMove={[Function]}
              onResponderRelease={[Function]}
              onResponderTerminate={[Function]}
              onResponderTerminationRequest={[Function]}
              onStartShouldSetResponder={[Function]}
              style={
                Array [
                  Object {
                    "backgroundColor": "transparent",
                  },
                  Object {
                    "flex": 1,
                  },
                ]
              }
              testID={undefined}
              tvParallaxProperties={undefined}
            >
              <View
                style={
                  Object {
                    "flex": 1,
                  }
                }
              />
            </View>
            <View
              accessibilityComponentType={undefined}
              accessibilityLabel={undefined}
              accessibilityTraits={undefined}
              accessible={true}
              hasTVPreferredFocus={undefined}
              hitSlop={undefined}
              isTVSelectable={true}
              nativeID={undefined}
              onLayout={undefined}
              onResponderGrant={[Function]}
              onResponderMove={[Function]}
              onResponderRelease={[Function]}
              onResponderTerminate={[Function]}
              onResponderTerminationRequest={[Function]}
              onStartShouldSetResponder={[Function]}
              style={
                Array [
                  Object {
                    "backgroundColor": "transparent",
                  },
                  Object {
                    "flex": 1,
                  },
                ]
              }
              testID={undefined}
              tvParallaxProperties={undefined}
            >
              <View
                style={
                  Object {
                    "flex": 1,
                  }
                }
              />
            </View>
          </View>
        </View>
        <View>
          <Text
            accessible={true}
            allowFontScaling={true}
            disabled={false}
            ellipsizeMode="tail"
            style={
              Object {
                "color": "white",
                "fontSize": 14,
                "fontWeight": "300",
              }
            }
          >
            { main: { text: 'thing' } }
          </Text>
          <View
            style={
              Object {
                "bottom": 0,
                "flexDirection": "row",
                "left": 0,
                "position": "absolute",
                "right": 0,
                "top": 0,
              }
            }
          >
            <View
              accessibilityComponentType={undefined}
              accessibilityLabel={undefined}
              accessibilityTraits={undefined}
              accessible={true}
              hasTVPreferredFocus={undefined}
              hitSlop={undefined}
              isTVSelectable={true}
              nativeID={undefined}
              onLayout={undefined}
              onResponderGrant={[Function]}
              onResponderMove={[Function]}
              onResponderRelease={[Function]}
              onResponderTerminate={[Function]}
              onResponderTerminationRequest={[Function]}
              onStartShouldSetResponder={[Function]}
              style={
                Array [
                  Object {
                    "backgroundColor": "transparent",
                  },
                  Object {
                    "flex": 1,
                  },
                ]
              }
              testID={undefined}
              tvParallaxProperties={undefined}
            >
              <View
                style={
                  Object {
                    "flex": 1,
                  }
                }
              />
            </View>
            <View
              accessibilityComponentType={undefined}
              accessibilityLabel={undefined}
              accessibilityTraits={undefined}
              accessible={true}
              hasTVPreferredFocus={undefined}
              hitSlop={undefined}
              isTVSelectable={true}
              nativeID={undefined}
              onLayout={undefined}
              onResponderGrant={[Function]}
              onResponderMove={[Function]}
              onResponderRelease={[Function]}
              onResponderTerminate={[Function]}
              onResponderTerminationRequest={[Function]}
              onStartShouldSetResponder={[Function]}
              style={
                Array [
                  Object {
                    "backgroundColor": "transparent",
                  },
                  Object {
                    "flex": 1,
                  },
                ]
              }
              testID={undefined}
              tvParallaxProperties={undefined}
            >
              <View
                style={
                  Object {
                    "flex": 1,
                  }
                }
              />
            </View>
          </View>
        </View>
      </View>
    </View>
  </RCTScrollView>
</View>
`;
